src/main/java/seedu/address/model/note/Note.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.note;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+
+public class Note {
+
+    //Identity field
+    private final Title title;
+
+    //Data field
+    private final Content content;
+    //tags to be added
+
+    public Note(Title title, Content content) {
+        requireAllNonNull(title, content);
+        this.title = title;
+        this.content = content;
+    }
+
+    public Title getTitle() {
+        return this.title;
+    }
+
+    public Content getContent() {
+        return this.content;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof Note)) {
+            return false;
+        }
+
+        Note otherNote = (Note) other;
+        return otherNote.getTitle().equals(getTitle())
+                && otherNote.getContent().equals(getContent());
+
+    }
```

Comment:
For the equals override we probably should also override hashcode as well. Since if two objects are equal, their hashcodes must be equal

```java
@Override
public int hashCode() {
    return Objects.hash(getTitle(), getContent());
}
```
Similar for all other serialisable classes like title, content and notebook, otherwise testcases may fail due to `model` object inequality.

Reply:
Got it, will update it

Reply:
@rui-han-crh 
Updated in [3b38efb](https://github.com/AY2223S1-CS2103T-W12-2/tp/pull/54/commits/3b38efb71a7930cdf5d275e8b4b6cbb992062c9e)

Thank you!